packages/flutter/lib/src/foundation/debug.dart

```diff
@@ -140,13 +140,15 @@ String? connectedVmServiceUri;
 /// [FlutterMemoryAllocations] should not increase size of the Flutter application
 /// if memory allocations are disabled.
 ///
+/// [flutterLibrary] is the name of the Flutter library where the object is declared.
```

Comment:
nit: Docs should be grammatically correct and start with an upper case character, see https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-correct-grammar